src/GeositeFramework/js/Plugin.js

```diff
@@ -32,7 +32,35 @@ require(['use!Geosite',
                 pluginName = model.get('pluginSrcFolder'),
                 $uiContainer = model.get('$uiContainer'),
                 $legendContainer = model.get('$legendContainer'),
-                logger = new Logger(pluginName);
+                logger = new Logger(pluginName),
+                resizers = {
+                    setWidth: function(val) {
+                        var d = $.Deferred(),
+                            setter = function(w) {
+                                return function() {
+                                    $uiContainer.css({ width: "" });
+                                    $uiContainer.removeClass(function() {
+                                        var classes = this.className.split(" ");
+                                        return _.filter(classes, function(c) {
+                                            return /sidebar-width-.*/.test(c);
+                                        }).join(" ");
+                                    });
+                                    $uiContainer.addClass("sidebar-width-" + w);
+                                };
+                            };
+                        _.delay(function() {
+                            if (typeof val === "string") {
+                                d.then(setter(val));
+                            } else if (typeof val === "number") {
+                                d.then(function() {
+                                    $uiContainer.css({ width: val });
+                                });
+                            }
+                            d.resolve();
+                        }, 1000);
```

Comment:
I understand the comment to add this, but I don't think we need to actually add an arbitrary delay of one second.  Let's cut this down to `100`, so that the class has been applied but not incur a noticeable lag.